libkbfs/block_types.go

```diff
@@ -62,6 +62,19 @@ func (cb *CommonBlock) DataVersion() DataVer {
 	return FirstValidDataVer
 }
 
+// NewEmpty implements the Block interface for CommonBlock
+func (cb *CommonBlock) NewEmpty() Block {
+	return NewCommonBlock()
+}
+
+// Set implements the Block interface for CommonBlock
+func (cb *CommonBlock) Set(other Block) {
```

Comment:
Yup I thought I did call that, but this is a very good case for making it reuse DeepCopy.